master/buildbot/www/rest.py

```diff
@@ -249,6 +249,15 @@ def checkFields(fields, negOk=False):
                 except Exception:
                     raise BadRequest('invalid offset')
                 continue
+            # As discussed with @tardyp, this is the best way to implement
+            # properties behavior.
+            elif arg in ('property',):
+                try:
+                    properties = [v.decode('utf-8') for v in reqArgs[arg]]
+                except Exception:
+                    raise BadRequest('invalid property value for %s' % arg)
+                filters.append(resultspec.Filter(arg, 'eq', properties))
```

Comment:
I would prefer if this is not sent as a filter.
It is more like an additional field rather than a filter